src/gui_common/CustomRichTextLabel.cs

```diff
@@ -632,4 +644,58 @@ private void OnMetaClicked(object meta)
             }
         }
     }
+
+    private void OnMetaHoverStarted(object meta)
+    {
+        if (!EnableTooltipsForMetaTags || speciesPreviewTooltipScene == null)
+            return;
+
+        if (meta is string metaString && metaString.StartsWith("species", StringComparison.Ordinal))
+        {
+            var speciesCode = GUICommon.ConvertFromSafeMetaTag(metaString.Split("species:")[1]);
+            var species = ThriveJsonConverter.Instance.DeserializeObject<Species>(speciesCode);
+
+            if (species is not MicrobeSpecies)
+                return;
+
+            var tooltip = ToolTipManager.Instance
+                .GetToolTipIfExists<SpeciesPreviewTooltip>(species.FormattedName, "speciesPreview");
+            if (tooltip == null)
+            {
+                tooltip = speciesPreviewTooltipScene.Instance<SpeciesPreviewTooltip>();
+                tooltip.Name = species.FormattedName;
+                ToolTipManager.Instance.AddToolTip(tooltip, "speciesPreview");
+            }
+
+            // Might need to refresh species if tooltip has previous generation stored or same name from previous game
+            if (tooltip.PreviewSpecies?.StringCode != speciesCode)
+                tooltip.PreviewSpecies = species;
+
+            ToolTipManager.Instance.MainToolTip = tooltip;
+            ToolTipManager.Instance.Display = true;
+        }
+    }
+
+    private void OnMetaHoverEnded(object meta)
+    {
+        if (!EnableTooltipsForMetaTags)
+            return;
+
+        if (meta is string metaString && metaString.StartsWith("species", StringComparison.Ordinal))
+        {
+            var speciesCode = GUICommon.ConvertFromSafeMetaTag(metaString.Split("species:")[1]);
+            var species = ThriveJsonConverter.Instance.DeserializeObject<Species>(speciesCode);
+
+            if (species is not MicrobeSpecies)
+                return;
+
+            var tooltip = ToolTipManager.Instance
+                .GetToolTipIfExists<SpeciesPreviewTooltip>(species.FormattedName, "speciesPreview");
+            if (ToolTipManager.Instance.MainToolTip == tooltip)
+            {
+                ToolTipManager.Instance.MainToolTip = null;
+                ToolTipManager.Instance.Display = false;
```

Comment:
Very possible I am getting something wrong, but It's the same as `DisplayName` here.